.drone.yml

```diff
@@ -1,17 +1,41 @@
+workspace:
+  base: /var/www
+  path: html
+
 services:
+  web:
+    image: fpfis/httpd-php-dev:${PHP_VERSION=5.6}
+    environment:
+    - DOCUMENT_ROOT=/var/www/html
   mysql:
-    image: fpfis/mysql56
+    image: percona/percona-server:5.6
```

Comment:
I am wondering if there is any restriction to not use a variable to hold default php version, avoiding repeating it all over.
Something like:
matrix:
  - PHP_DEFAULT: 5.6

Reply:
I mostly copy paste the file from the OpenEuropa components. 
I don't know if this is really necessary to have it in a variable. Could you explain a bit why it would be interesting? Thx